### PR TITLE
[FIX] account & l10n_ch: fix l10n_ch installation on a company

### PR DESCRIPTION
**Pull request by FlorianGilbert**
--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

**Comment by FlorianGilbert** [comment]
@csnauwaert

**Comment by FlorianGilbert** [comment]
Wait, I have found another bug on l10n_ch, I will fix on this branch !

**Comment by FlorianGilbert** [comment]
It is fixed. Ok for a review @csnauwaert

**Comment by csnauwaert** [comment]
Seems good to me

File: addons/l10n_ch/__init__.py [hunk 1]
```diff
@@ -14,9 +14,7 @@ def init_settings(env):
     '''If the company is localized in Switzerland, activate the cash rounding by default.
     '''
     # The cash rounding is activated by default only if the company is localized in Switzerland.
-    for company in env['res.company'].search([]):
-        if company.country_id != env.ref('base.ch'):
-            continue
+    for company in env['res.company'].search([('partner_id.country_id', '=', 'ch')]):
```
Comment (csnauwaert) [inline addons/l10n_ch/__init__.py]: actually, are you sure this is working?
Would it not be better to have smth like this:
```
ch_country = env.ref('base.ch')
for company in env['res.company'].search([('partner_id.country_id', '=', ch_country)]):
```

Comment (FlorianGilbert) [reply addons/l10n_ch/__init__.py]: Yep it's working, I have test your solution and it's working too but with a little modification:
`
ch_country = env.ref('base.ch')
for company in env['res.company'].search([('partner_id.country_id', '=', ch_country.id)]):
`
We need to put **.id** after the `ch_country` but it's work. I will push the modification now ! 
Thanks

**Comment by csnauwaert** [comment]
@robodoo r+

**Pull request closed without merging**